src/test/java/com/hedera/exchange/ExchangeRateUtilsTestCases.java

```diff
@@ -68,4 +108,284 @@ public void getDecryptedEnvironmentVariableFromAWSTest() {
 		final String lambdaFunctioName = "exchange-rate-tool-lambda-integration";
 		assertEquals(expectedValue, ExchangeRateUtils.getDecryptedValueFromAWS(encryptedValue, lambdaFunctioName));
 	}
+
+	@Test
+	public void generateExchangesTest() {
+		//setup
+		Map<String, String> exchangeAPIs = new HashMap<String, String>() {{
+			put("bitrex", "https://api.bittrex.com/api/v1.1/public/getmarketsummary?market=USD-HBAR");
+			put("okcoin", "https://www.okcoin.com/api/spot/v3/instruments/HBAR-USD/ticker");
+			put("paybito", "https://trade.paybito.com/api/trades/HBAR_USD");
+		}};
+
+		//when
+		List<Exchange> exchanges = ExchangeRateUtils.generateExchanges(exchangeAPIs);
+
+		//then
+		assertEquals(3, exchanges.size());
+		assertNotNull(exchanges.get(0).getHBarValue());
+		assertNotNull(exchanges.get(1).getHBarValue());
+		assertNotNull(exchanges.get(2).getHBarValue());
+	}
+
+	@Test
+	public void testcalculateNewExchangeRate() throws IOException {
+		//given
+		Rate activeRate = new Rate(30000 ,960000, 123456789L);
```

Comment:
for numbers, use the underscore `_` to split the value, i.e., `30000 -> 30_000`, `960000 -> 960_000`, etc